ister_gui.py

```diff
@@ -2085,14 +2085,17 @@ def handler(self, config):
     def run_ui(self):
         return self._ui.do_form()
 
-    def build_ui_widgets(self):
+    def build_ui_widgets(self, config):
         desc = urwid.Text('console=tty0 console=ttyS0,115200n8 quiet '
                           'init=/usr/lib/systemd/systemd-bootchart '
                           'initcall_debug tsc=reliable no_timer_check '
                           'noreplace-smp kvm-intel.nested=1 '
                           'rootfstype=ext4,btrfs,xfs intel_iommu=igfx_off '
                           'cryptomgr.notests rcupdate.rcu_expedited=1 '
                           'i915.fastboot=1  rw rootwait')
+        for dev_entry in config['PartitionMountPoints']:
+            if 'encryption' in dev_entry:
+                self.cmdline_field.set_edit_text("rootflags=x-systemd.mount-timout=infinity,retry=10000")
```

Comment:
What is retry doing here? All I saw on this was that setting rd.luks.options=timeout=0 rootflags=x-systemd.device-timeout=0 would disable the timeout for entering a password.

Reply:
the retry is for mount root, but I will keep it as we have in clr-installer, updating...